blog/2015/10/call-for-instructor-training.html

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog
+root: ../../..
+author: Raniere Silva
+title: "Call for Applications to December Instructor Training"
+date: 2015-10-20
+time: "22:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+As we <a href="{{page.root}}/blog/2015/09/rebooting-instructor-training.html">announced previous</a>,
+we will run another round of the free/open Instructor Training in December.
+Is time for people interested in participate at this round apply for the
```

Comment:
People who are interested in participating in this round can now apply to take part.